src/hott/01-paths.rzk.md

```diff
@@ -194,15 +194,15 @@ Concatenation of two paths with common domain; defined using `#!rzk concat` and
 
 ```rzk
 #def zag-zig-concat
-  (p : y = x)
-  (q : y = z)
-  : (x = z)
+  ( p : y = x)
+  ( q : y = z)
+  : ( x = z)
   := concat A x y z (rev A y x p) q
 
 #def right-cancel-concat
   ( p q : x = y)
   ( r : y = z)
-  : ((concat A x y z p r) = (concat A x y z q r)) → (p = q)
+  : ( ( concat A x y z p r) = (concat A x y z q r)) → (p = q)
```

Comment:
Here I don't think spaces should be inserted, but doing so is fine as the parentheses shouldn't actually be there in my opinion.

Reply:
Ah, cool! If you don't mind, maybe I can share one specific opinion on the matter. Namely, I'd be careful to make the linter too obnoxious about "redundant" parentheses. At times, parentheses may be redundant in the sense that the code parses the same way to the typechecker without them, but they may still convey useful information to the reader of the code. For instance, if you have defined concatenation of identifications to be left associative, then you could formulate the associator to have type
```text
  p ∙ q ∙ r = p ∙ (q ∙ r),
```
but this looks rather ridiculous. Of course, in this hypothetical scenario, you want to properly parenthesize this expression as
```text
  (p ∙ q) ∙ r = p ∙ (q ∙ r).
``` 
Personally, I would find it quite annoying if the linter told me that this parenthesis was redundant, because it isn't.

Reply:
That's a good point and we should definitely consider. Once the linter is added to the backlog, we will surely ask for feedback on what rules it should or shouldn't have